### ISSUE TITLE
Interpolation

### ISSUE THREAD

**Issue by nbodagh** [CemrgAppDevelopers/CemrgApp#66]
I'm having an issue CEMRG. When I try to segment and change the interpolation to 3D the program crashes

Have you ever had anyone have this before?

Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.LogLevel=2
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:215
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.CacheSize=30
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:217
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.ThreadPoolSize=20
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:219
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.Timeout=5000
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:221
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.RequireTopic=true
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:223
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.LogLevel=2
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:215
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.CacheSize=30
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:217
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.ThreadPoolSize=20
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:219
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.Timeout=5000
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:221
Wed Jul 6 14:47:19 2022 Debug: org.commontk.eventadmin.RequireTopic=true
in ctkEAConfiguration::startOrUpdate, C:\dev\build\CEMRGapp\ep\src\CTK\Plugins\org.commontk.eventadmin\ctkEAConfiguration.cpp:223
Wed Jul  6 14:47:19 2022
3.14 core.mod.core.log: Logfile: C:/Users/nboda/AppData/Local/KCL/CemrgApp v2.1_1584152459/data/11/mitk-0.log
3.38 ERROR: BlueBerry Workbench not running!
3.85 BlueBerry WARNING: Could not get plug-in resource: CemrgAppQtHelpCollection.qhc

**Comment by OrodRazeghi** [comment]
@nbodagh just to find the root cause, were you actively segmenting the image, when the 3D interpolation was on? or the crash happened after you were done with the segmentation and just wanted to see the final result?

**Comment by nbodagh** [comment]
I had finished the segmentation and wanted to see the final result

**Comment by nbodagh** [comment]
Now whenever I try to change it to 3D before I start segmenting, the programme crashes

**Comment by nbodagh** [comment]
Hi, just writing to ask - do you have any idea about what could be causing this?

**Comment by OrodRazeghi** [comment]
What is the file format for the image you are trying to segment? nifti? What is the resolution of your image? Is it a really high-res image?

**Comment by nbodagh** [comment]
Yes, it was nifti and they were high res images. Its strange because it was letting me do it before

**Comment by OrodRazeghi** [comment]
try downsampling the image to a lower res, and see if it resolves the issue. If you can, please also try segmenting directly on the MITK [workbench](https://www.mitk.org/wiki/Downloads), and let us know if you face a similar crash.

**Comment by nbodagh** [comment]
Hi, sorry I'm fairly new to CEMRG. How do I do both of those?

**Comment by OrodRazeghi** [comment]
You don't have to use CemrgApp to do the resampling, you can use ITKSnap or any similar image processing software that you are familiar with, if you want to use CemrgApp, there is a basic image processing [plugin](https://docs.mitk.org/nightly/org_mitk_views_basicimageprocessing.html), which you can use for resampling images.
One important detail is that the image you are segmenting should be a nifti with the `.nii` extension.
Some docs and tutorials on using the segmentation tools in the scar plugin: [sop](https://cemrg.com/software/CemrgApp-AtrialScarProcessing.pdf), [video](https://cemrg.com/software/scar-video1.mp4).
